khi_robot_control/CMakeLists.txt

```diff
@@ -47,10 +47,8 @@ else()
   set(krnx_lib_name krnx-x86_64)
 endif()
 
-add_library(krnx_lib SHARED IMPORTED)
-set_target_properties(krnx_lib PROPERTIES
-  IMPORTED_LOCATION ${CMAKE_CURRENT_LIST_DIR}/lib/lib${krnx_lib_name}.so)
-set(krnx_LIBRARIES krnx_lib)
+find_library(krnx_LIBRARIES lib${krnx_lib_name}.so PATHS ${CMAKE_SOURCE_DIR}/lib)
+message(STATUS "krnx LIBRARIES found : ${krnx_LIBRARIES}")
```

Comment:
Can you clarify why you use `find_library(..)` here? The `lib` directory contains these libraries, and we know they are there, so looking for them seems like it would be unnecessary.